### ISSUE TITLE
Proof of Concept(POC) for hugging faces

### ISSUE THREAD

**Issue by aadilvagh** [amitbhudiya/CITS5206_DialogCoder#5]
Choose "distilbert-base-uncased" model for offline operation, using recommended parameters for the first shot.
The model outputs the top 5 labels with confidence scores, but these scores are currently only in the range of 0.3 to 0.7.

**Comment by whatthehiao** [comment]
1. For now, the training data is relatively small (427 rows csv file). It would be ideal to have 1k+ rows of data to improve the model. Maybe we can request more data from the client.
2. Fine-tune the recommended parameters until we find the optimal combination for improved accuracy.
3. Consider other transformer models, like BERT? while keep the model size limit of like, 2GB? for portability.
